spec/e2e/features/short_form/error_messages.feature

```diff
@@ -32,6 +32,14 @@ Feature: Short Form Application
 
         When I fill out the Contact page with an address (non-NRHP match) and WorkInSF
         And I confirm my address
+        # error: invalid email in alternate contact
+        And I fill out the AlternateContact page with the email "grant@exygy"
+        Then I should see an email error on the AlternateContact page
+
+        When I navigate to the "You" section
```

Comment:
This seems a bit redundant to navigate back to the You page again, then skip it, but the alternate contact thing is weird because there are multiple pages so it's not as easy to refill this page. 

Let me know if you have any better suggestions